.github/workflows/deploy-services-sandbox.yml

```
@@ -0,0 +1,81 @@
+name: Deploy Service's Test Sandbox Stack to AWS
+
+on:
+  push:
+    branches:
+      - services-hyp3-sandbox
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - environment: hyp3-services-test
+            domain: hyp3-services-test.asf.alaska.edu
```

Note that this will require migrating the current [`hyp3-services-test`](https://github.com/ASFHyP3/hyp3/settings/environments/5575309217/edit) GitHub environment to a `hyp3-services-sandbox` environment. I don't think there's a way to rename GitHub environment, unfortunately!
```suggestion
          - environment: hyp3-services-sandbox
            domain: hyp3-services-sandbox.asf.alaska.edu
```